src/appsco-audit-log-page.html

```diff
@@ -713,10 +713,15 @@
                     }.bind(this));
 
                     this.set('_eventTypeListDisplay', this._eventTypeList);
-
+                    this._setDefaultEventType();
                 }
             },
 
+            _setDefaultEventType: function() {
+                this.$.appscoSearchEventType.setValue('All event types');
```

Comment:
Instead of string 'All event types' set this from property named _eventTypeList. Get its first item->text.